### ISSUE TITLE
Remove Legacy Code

### ISSUE THREAD

**Issue by Zauberfisch** [swipestripe/silverstripe-swipestripe#38]
There are a couple of legacy code snippets in the core, which we should remove.

examples:
- LicenseKey db field
- some left over of Stock management

**Comment by frankmullenger** [comment]
Yeah, I think we discussed the stock mgmt code on skype last night but either way happy for it to go.

**Comment by Zauberfisch** [comment]
actually created this issue while we talked ;)
Its more of a reminder so we don't forget to do it.
